src/uagents/asgi.py

```diff
@@ -37,7 +37,9 @@ def __init__(
         self._queries = queries
 
     async def serve(self):
-        config = uvicorn.Config(self, host="0.0.0.0", port=self._port, log_level="info")
+        config = uvicorn.Config(
```

Comment:
I think is good to get rid of the extra "status 200" messages, but I'm wondering if we should add some extra startup and shutdown logging to replace what uvicorn was doing? Something like:
```
INFO:     Running startup tasks.
INFO:     Startup complete.
INFO:     Server running on http://0.0.0.0:8000 (Press CTRL+C to quit)
...
INFO:     Running shutdown tasks.
INFO:     Shutdown complete.
```